src/components/Book.js

```diff
@@ -1,12 +1,62 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+import { deleteBook } from '../redux/books/books';
+import './Book.css';
 
 export default function Book(props) {
   const book = props;
+  const dispatch = useDispatch();
+  const remove = () => {
+    dispatch(deleteBook(book.id));
+  };
+  const chapterVal = Math.round(Math.random() * 20);
+
   return (
     <div className="book">
-      <div className="book-title">{book.title}</div>
-      <div className="book-authors">{book.author}</div>
-      <button type="button">Remove</button>
+      <div className="ctn-left">
+        <div className="ctn-book-info">
+          <h5 className="category">{book.category}</h5>
+          <h2 className="title">{book.title}</h2>
+          <h5 className="author">{book.author}</h5>
+        </div>
+        <div className="ctn-book-action">
+          <h4>Comments</h4>
+          <span>|</span>
+          <button type="button" onClick={() => remove(book.id)}>
+            Remove
+          </button>
+          <span>|</span>
+          <h4>Edit</h4>
+        </div>
+      </div>
+
+      <div className="ctn-middle">
+        <div>
+          <image alt={book.title} />
+        </div>
+        <div className="state-progress">
+          <div style={{ width: 50, height: 50 }}>
```

Comment:
- [ ]  Inline styling is one of CSS bad practices. When you add inline styles, it makes it difficult revising the design of your site and makes your code harder to manage.
I would recommend moving the style out of the `div`. Try to define a constant `style` in the `Book` component and give it to the style attribute in the div.